recipes/liblsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+# Conda build recipe for liblsl
+
+{% set name = "liblsl" %}
+{% set version = "1.14.1b8" %}
```

Comment:
This looks like a pre-release. Can you change it to an actual release?

Reply:
Of course. There were some changes needed for the conda environment (mostly related to RPATH handling), so my idea was to iron out all the conda-related edges, release the next version with the fixes included and then change the version in the `meta.yaml`.